UITests/Global.swift

```diff
@@ -162,6 +162,37 @@ extension KIFUITestActor {
             return stepResult
         }
     }
+    
+    /**
+     * Long presses a WKWebView element with the given label, for a given duration.
+     */
+    func longPressWebViewElementWithAccessibilityLabel(text: String, duration: NSTimeInterval) {
+        let webView = getWebViewWithKIFHelper()
+        let stepResult = MutableBox(KIFTestStepResult.Wait)
+        
+        let escaped = text.stringByReplacingOccurrencesOfString("\"", withString: "\\\"")
+        webView.evaluateJavaScript("KIFHelper.longPressElementWithAccessibilityLabel(\"\(escaped)\", \(duration * 1000))") { success, _ in
+            if success as? Bool == false {
+                stepResult.value =  KIFTestStepResult.Failure
+            } else {
+                NSTimer.scheduledTimerWithTimeInterval(duration, target: self, selector: #selector(self.didFinishWaitingForLongPressToComplete), userInfo: stepResult, repeats: false)
```

Comment:
Nit: Instead of having to wrap this in a `MutableBox` instance, you can use a `dispatch_timer` block instead;

```swift
dispatch_after(1000 * duration, dispatch_get_main_queue()) {
    stepResult.value = KIFTestStepResult.Success
}
```